weni_cli/validators/definition.py

```diff
@@ -246,15 +244,115 @@ def validate_agent_definition_schema(data):
 
                     # If contact_field is True, validate parameter name
 
-                    if param_data.get("contact_field") and not ContactFieldValidator.has_valid_contact_field_name(param_name):
+                    if param_data.get("contact_field") and not ContactFieldValidator.has_valid_contact_field_name(
+                        param_name
+                    ):
                         return f"Agent '{agent_key}': tool '{tool_name}': parameter '{param_name}' name must match the regex of a valid contact field: {re.escape(ContactFieldValidator.CONTACT_FIELD_NAME_REGEX)} in the agent definition file"
 
-                    if param_data.get("contact_field") and not ContactFieldValidator.has_valid_contact_field_length(param_name):
+                    if param_data.get("contact_field") and not ContactFieldValidator.has_valid_contact_field_length(
+                        param_name
+                    ):
                         return f"Agent '{agent_key}': tool '{tool_name}': parameter '{param_name}' name must be {ContactFieldValidator.CONTACT_FIELD_MAX_LENGTH} characters or less in the agent definition file"
 
-                    if param_data.get("contact_field") and not ContactFieldValidator.has_allowed_parameter_name(param_name):
+                    if param_data.get("contact_field") and not ContactFieldValidator.has_allowed_parameter_name(
+                        param_name
+                    ):
                         return f"Agent '{agent_key}': tool '{tool_name}': parameter '{param_name}' name must not be a reserved contact field name in the agent definition file\nRestricted contact field names: {ContactFieldValidator.RESERVED_CONTACT_FIELDS}"
 
+
+def validate_active_agent_definition_schema(data):
+    # Check that agents section exists
+    if data.get("agents") is None:
+        return "Missing required root key 'agents' in the agent definition file"
+
+    if not isinstance(data["agents"], dict):
+        return "'agents' must be an object in the agent definition file"
+
+    if not data["agents"]:
+        return "No agents defined in the agent definition file"
+
+    # For each agent, validate required fields
+    for agent_key, agent_data in data["agents"].items():
+        # Check required agent fields
+        if not isinstance(agent_data, dict):
+            return f"Agent '{agent_key}' must be an object in the agent definition file"
+
+        # Validate webhook_example if present (must be a list with at least one element)
+        if "webhook_example" in agent_data:
```

Comment:
Webhook example does not exist anymore, is now `result_examples_file` and is placed inside the `pre-processing` key

Reply:
It is now a simple string too, not a list